tests/main/interfaces-many-core-provided/task.yaml

```diff
@@ -104,4 +113,13 @@ execute: |
                 echo "skipping $plugcmd on an unsupported system"
             fi
         fi
+
+        echo "Finally disconnect the interface"
+        if snap interfaces | grep -E -q "$CONNECTED_PATTERN"; then
+            if [ "$plug_iface" = "$CONSUMER_SNAP:browser-sandbox" ]; then
+                snap disconnect "$CONSUMER_SNAP:browser-support" "$slot_iface"
+            else
+                snap disconnect "$plug_iface" "$slot_iface"
+            fi
+        fi
```

Comment:
This changes the way we disconnect all interfaces in this test (previously they were disconnected on snap remove) and affects all systems, not just debian. Is there a reason to change this part of the test?

Reply:
@stolowski, thanks for the review, I did that because if there is a problem while disconnecting an interface we can know which is the interface, otherwise we see an error uninstalling a snap. I think is more clear if we connect/disconnect.

Reply:
Ah, I see, makes sense, thanks.